tests/test_plotting.py

```diff
@@ -35,404 +35,543 @@
 tol_3d = 15
 
 
+@pytest.mark.unit
 def test_kwarg_warning(DummyStellarator):
+    """Test that passing in unknown kwargs throws an error."""
     eq = Equilibrium.load(load_from=str(DummyStellarator["output_path"]))
     with pytest.raises(AssertionError):
         fig, ax = plot_1d(eq, "p", not_a_kwarg=True)
     return None
 
 
+@pytest.mark.unit
+@pytest.mark.solve
 @pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_1d)
 def test_1d_p(SOLOVEV):
+    """Test plotting 1d pressure profile."""
     eq = EquilibriaFamily.load(load_from=str(SOLOVEV["desc_h5_path"]))[-1]
     fig, ax = plot_1d(eq, "p", figsize=(4, 4))
     return fig
 
 
+@pytest.mark.unit
+@pytest.mark.solve
 @pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_1d)
-def test_1d_dpdr(DSHAPE):
-    eq = EquilibriaFamily.load(load_from=str(DSHAPE["desc_h5_path"]))[-1]
+def test_1d_dpdr(DSHAPE_current):
+    """Test plotting 1d pressure derivative."""
+    eq = EquilibriaFamily.load(load_from=str(DSHAPE_current["desc_h5_path"]))[-1]
     fig, ax = plot_1d(eq, "p_r", figsize=(4, 4))
     return fig
 
 
+@pytest.mark.unit
+@pytest.mark.solve
 @pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_1d)
-def test_1d_iota(DSHAPE):
-    eq = EquilibriaFamily.load(load_from=str(DSHAPE["desc_h5_path"]))[-1]
+def test_1d_iota(DSHAPE_current):
+    """Test plotting 1d rotational transform."""
+    eq = EquilibriaFamily.load(load_from=str(DSHAPE_current["desc_h5_path"]))[-1]
     grid = LinearGrid(rho=0.5, theta=100, zeta=0.0)
     fig, ax = plot_1d(eq, "iota", grid=grid, figsize=(4, 4))
     return fig
 
 
+@pytest.mark.unit
+@pytest.mark.solve
 @pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_1d)
-def test_1d_logpsi(DSHAPE):
-    eq = EquilibriaFamily.load(load_from=str(DSHAPE["desc_h5_path"]))[-1]
+def test_1d_logpsi(DSHAPE_current):
+    """Test plotting 1d flux funciton with log scale."""
+    eq = EquilibriaFamily.load(load_from=str(DSHAPE_current["desc_h5_path"]))[-1]
     fig, ax = plot_1d(eq, "psi", log=True, figsize=(4, 4))
     ax.set_ylim([1e-5, 1e0])
     return fig
 
 
+@pytest.mark.unit
+@pytest.mark.solve
 @pytest.mark.mpl_image_compare(remove_text=True, tolerance=10)
-def test_2d_logF(DSHAPE):
-    # plot test is inconsistent
-    eq = EquilibriaFamily.load(load_from=str(DSHAPE["desc_h5_path"]))[-1]
+def test_2d_logF(DSHAPE_current):
+    """Test plotting 2d force error with log scale."""
+    eq = EquilibriaFamily.load(load_from=str(DSHAPE_current["desc_h5_path"]))[-1]
     grid = LinearGrid(rho=100, theta=100, zeta=0.0)
     fig, ax = plot_2d(eq, "|F|", log=True, grid=grid, figsize=(4, 4))
     return fig
 
 
+@pytest.mark.unit
+@pytest.mark.solve
 @pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_2d)
-def test_2d_g_tz(DSHAPE):
-    eq = EquilibriaFamily.load(load_from=str(DSHAPE["desc_h5_path"]))[-1]
+def test_2d_g_tz(DSHAPE_current):
```

Comment:
the jacobian of the new current-solved DSHAPE seems the opposite sign of the old one? not really a problem just asking as the color scheme seems inverted

Reply:
Yeah it's due to some sign issues. The regular `DSHAPE` example should give positive current, but because of the sign issues (see #202) it's negative. So the `DSHAPE_current` example has negative input current, which then gives negative iota, flipping the helicity compared to the regular `DSHAPE`